### PR TITLE
Added support for Image,Video and Doc File size.

### PR DESCRIPTION
**Pull request by Rajnish23**
@DroidNinja Please Review the changes!!

### CODE REVIEW

**Comment by Rajnish23** [comment]
@DroidNinja Please Review the PR and let me know if any modification required.

File: filepicker/src/main/java/droidninja/filepicker/FilePickerBuilder.kt [hunk 1]
```diff
@@ -22,6 +22,21 @@ class FilePickerBuilder {
 
     private val mPickerOptionsBundle: Bundle = Bundle()
 
+    fun setImageFileSize(fileSize: Int): FilePickerBuilder {
```
Comment (DroidNinja) [inline filepicker/src/main/java/droidninja/filepicker/FilePickerBuilder.kt:25]: Can you change this name to `setImageSizeLimit` and `setVideoSizeLimit`?

**Comment by DroidNinja** [comment]
@Rajnish23 PR looks good. Please do the required changes and change the destination PR branch to `develop`. Will merge this with other enhancements. Thanks!!

File: filepicker/src/main/java/droidninja/filepicker/FilePickerBuilder.kt (hunk 1 above)
Comment (Rajnish23) [reply filepicker/src/main/java/droidninja/filepicker/FilePickerBuilder.kt:25]: Sure, I've made the changes.

**Comment by Rajnish23** [comment]
@DroidNinja Made the required changes also changed the destination PR to `develop` branch.

**Review by DroidNinja — APPROVED** [review]
Looks fine to me.

**Comment by Rajnish23** [comment]
> Looks fine to me.

Thank you.
Could you please add a hacktoberfest-accepted label to this PR.

**Pull request merged**